twilio/twiml/__init__.py

```diff
@@ -9,6 +10,21 @@ def lower_camel(string):
     return result[0].lower() + result[1:]
 
 
+def format_language(language):
+    """
+    Attempt to format language parameter as 'ww-WW'.
+
+    :param string language: language parameter
+    """
+    if not language:
+        return language
+
+    if not re.match('^\w\w[_-]\w\w$', language):
```

Comment:
Fair.  Since we're not validating that the string is an actual language specifier that's supported, we'd still be relying on the API to respond with an error if someone input something that was slightly wrong.  The regex was mostly there to check that things look mostly right.  Accepting digits and underscores didn't seem to be too far removed from the goal of just being a bit helpful.